### DISCUSSION TITLE
Add the ability to easily import/export labels, reports, and other resources

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5791] [Feature Requests]
https://discourse.specifysoftware.org/t/import-a-label-report/331

![image](https://user-images.githubusercontent.com/37256050/234083387-96910027-ea9b-4a64-8cfc-f4db49c8c2bf.png)
![image](https://user-images.githubusercontent.com/37256050/234083404-27c45739-f1a3-4228-88e8-86976f5cf6fa.png)

This bundles together both the query and the report resource into a format that can be easily imported and exported.

**Comment by maxpatiiuk** [comment]
From @grantfitzsimmons:

> ![image](https://user-images.githubusercontent.com/37256050/234077720-61a48226-d4d3-431b-825d-e0d4239baace.png)
>
> https://discourse.specifysoftware.org/t/importing-and-exporting-a-pick-list/285

**Comment by grantfitzsimmons** [comment]
I thnk these should be separate as they would be prioritized separaretly

**Comment by maxpatiiuk** [comment]
> I thnk these should be separate as they would be prioritized separaretly

I was thinking it would be one tool where support for more and more tables is incrementally added (depending on priorities)

**Comment by grantfitzsimmons** [comment]
Pick list import/export was requested by University of Michigan today

**Comment by mcruz-umich** [comment]
... and Schema import too :-)

**Comment by grantfitzsimmons** [comment]
@mcruz-umich https://github.com/specify/specify7/issues/1315

Yes, please!
